x-pack/plugins/security_solution/public/flyout/right/components/host_entity_overview.tsx

```diff
@@ -175,7 +175,7 @@ export const HostEntityOverview: React.FC<HostEntityOverviewProps> = ({ hostName
         description: (
           <>
             {hostRiskData ? (
-              <RiskScore severity={hostRiskData.host.risk.calculated_level} />
+              <RiskScoreLevel severity={hostRiskData.host.risk.calculated_level} />
```

Comment:
could you add the "learn more about risk score" link here as well?

Reply:
Thanks for spotting the inconsistency.